### ISSUE TITLE
ace interaction menu - modded vehicles - xeh_enable_auto -cba

### ISSUE THREAD

**Issue by Nagrach** [acemod/ACE3#2555]
ACE3 Version: 3.3.1
Stable

**Mods:**
@CBA_A3
@ace
@FA18 Superhornet

**Placed ACE3 Modules:**
No Factor

**Description:**
Missing Ace interaction menu on modded vehicles (FA18/Nato Equipment and others)
[BIS_fnc..... yaddayadda screenshot -> http://imgur.com/QKhQ3Cg

**Steps to reproduce:**
Start editor 
place playable unit
place modded vehicle (fa18 in this case)
get an consol error with warning about xeh auto enabler (screenshot)
try to interact with vehicle (cargo or other functions)

**Where did the issue occur?**
Editor

**RPT log file:**
http://pastebin.com/5pDbKLyv

We tested several mod vehicles with the new ACE3 3.3.1 and we are getting this error on some of them.
As crosscheck we tested it with BWMod Leo and no error occurred.
With the additional CBA enable auto - the error still shows up but the interaction menu is working but without the cargo functions - so only passengers stuff can be done.

Looks like a messy thingy between modmakers /cba/ace cargo.

**Comment by PabstMirror** [comment]
The mod has this in it's config:
```
		class Eventhandlers: Eventhandlers
		{
			init = "_this execVM ""\js_jc_FA18\scripts\init_FA18E.sqf""";
		};
```
This replaces CBA's init line, so it will break anything that uses `Extended_Init_EventHandlers`, such as ACE.  Adding the auto.pbo should fix all problems with this.

It's a known issue that the error can still show with the auto.pbo (fixed via 79d985a64883e2ad3c97e0799267175a8661d773).

For jets:
```
    class Plane: Air {
        GVAR(space) = 0;
        GVAR(hasCargo) = 0;
    };
```
So they can't actually use cargo (the vanilla like the A-10 are the same).

**Comment by PabstMirror** [comment]
The warning showing up when it shouldn't will be fixed in next release.

I believe cargo for attack jets is working as intended, we will want to look at changing it for transport planes (eg. C130).
